package.json

```diff
@@ -20,7 +20,7 @@
     "@linthtml/linthtml": "^0.5.1",
     "@mate-academy/eslint-config": "*",
     "@mate-academy/linthtml-config": "0.0.1",
-    "@mate-academy/scripts": "^0.7.12",
+    "@mate-academy/scripts": "^1.2.3",
```

Comment:
<img width="256" alt="image" src="https://user-images.githubusercontent.com/38065883/173184618-6d7fd159-61b3-42b3-9838-75c5c248d6fd.png">
Structure should be like this:

```html
<ul>
  <li>
    <ul>
      <li></li>
    </ul>
  </li>
  <li>
    <ul></ul>
  </li>
</ul>
```